resourcemanager/util/mock_gpool.go

```diff
@@ -38,7 +38,7 @@ func (*MockGPool) Tune(_ int) {
 
 // LastTunerTs is only for test
 func (*MockGPool) LastTunerTs() time.Time {
-	panic("implement me")
+	return time.Now().Add(-1 * 10 * time.Second)
```

Comment:
Why here is -1？

Reply:
The last Tuner timestamp must be before now.

Reply:
BTW, the api of time is in golang .


[func (t Time) Add(d Duration) Time](https://pkg.go.dev/time#Time.Add)
func (t [Time](https://pkg.go.dev/time#Time)) Sub(u [Time](https://pkg.go.dev/time#Time)) [Duration](https://pkg.go.dev/time#Duration)